### PR TITLE
[DO NOT MERGE] Debug Nemotron CI for release

### PR DESCRIPTION
**Pull request by b8zhong**
**Note: since this PR is still failing, it seems not related to this revert**

Debug-only PR to reproduce the Nemotron-3-Super NVFP4 + MTP NaN (`verify: target model logits`) on real CI hardware against `release/v0.5.13`.

Cannot reproduce locally (likely arch mismatch: local B300/sm_103 vs CI B200/sm_100). Using `/rerun-test` to trigger the registered runners.

**Do not merge.** Single no-op comment change only.

---
### CI States

Latest PR Test (Base): :x: [Run #27280504008](https://github.com/sgl-project/sglang/actions/runs/27280504008)
Latest PR Test (Extra): :x: [Run #27280504438](https://github.com/sgl-project/sglang/actions/runs/27280504438)

### CODE REVIEW

**Comment by b8zhong** [comment]
/rerun-test test/registered/8-gpu-models/test_nvidia_nemotron_3_super_nightly.py test/registered/4-gpu-models/test_nvidia_nemotron_3_super_nvfp4.py
